frontend/app/utils/locale-utils.ts

```diff
@@ -52,6 +53,7 @@ export async function initI18n(namespaces: Array<string>) {
     .init({
       appendNamespaceToMissingKey: true,
       debug: I18NEXT_DEBUG,
+      defaultNS: 'common',
```

Comment:
I feel like this should either be `defaultNS: false` or `defaultNS: 'gcweb'`. Preferrably `false`.

Eventually we will git rid of the `common` namespace and replace it with something that is a little more descriptive.

Reply:
I set `defaultNS: false`